internal/resource/config/localidentity/local_identity_identity_profiles_resource.go

```diff
@@ -0,0 +1,917 @@
+	readLocalIdentityIdentityProfilesResponse(ctx, apiReadLocalIdentityIdentityProfiles, &state)
+
+	// Set refreshed state
+	diags = resp.State.Set(ctx, &state)
+	resp.Diagnostics.Append(diags...)
+
+}
+
+// Update updates the resource and sets the updated Terraform state on success.
+func (r *localIdentityIdentityProfilesResource) Update(ctx context.Context, req resource.UpdateRequest, resp *resource.UpdateResponse) {
+	// Retrieve values from plan
+	var plan localIdentityIdentityProfilesResourceModel
+	diags := req.Plan.Get(ctx, &plan)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+	// Get the current state to see how any attributes are changing
+	var state localIdentityIdentityProfilesResourceModel
+	req.State.Get(ctx, &state)
+
+	updateLocalIdentityIdentityProfiles := r.apiClient.LocalIdentityIdentityProfilesApi.UpdateIdentityProfile(config.ProviderBasicAuthContext(ctx, r.providerConfig), plan.Id.ValueString())
+	apcId := plan.ApcId.Attributes()["id"].(types.String).ValueString()
+	apcResourceLink := client.NewResourceLink(apcId)
+	createUpdateRequest := client.NewLocalIdentityProfile(plan.Name.ValueString(), *apcResourceLink)
+	err := addOptionalLocalIdentityIdentityProfilesFields(ctx, createUpdateRequest, plan)
+	if err != nil {
+		resp.Diagnostics.AddError("Failed to add optional properties to add request for LocalIdentityIdentityProfiles", err.Error())
+		return
+	}
+	requestJson, err := createUpdateRequest.MarshalJSON()
+	if err == nil {
+		tflog.Debug(ctx, "Update request: "+string(requestJson))
+	}
+	updateLocalIdentityIdentityProfiles = updateLocalIdentityIdentityProfiles.Body(*createUpdateRequest)
+	updateLocalIdentityIdentityProfilesResponse, httpResp, err := r.apiClient.LocalIdentityIdentityProfilesApi.UpdateIdentityProfileExecute(updateLocalIdentityIdentityProfiles)
+	if err != nil {
+		config.ReportHttpError(ctx, &resp.Diagnostics, "An error occurred while updating LocalIdentityIdentityProfiles", err, httpResp)
+		return
+	}
+	// Log response JSON
+	responseJson, err := updateLocalIdentityIdentityProfilesResponse.MarshalJSON()
+	if err == nil {
+		tflog.Debug(ctx, "Read response: "+string(responseJson))
+	}
+	// Read the response
+	readLocalIdentityIdentityProfilesResponse(ctx, updateLocalIdentityIdentityProfilesResponse, &state)
+
+	// Update computed values
+	diags = resp.State.Set(ctx, state)
+	resp.Diagnostics.Append(diags...)
+}
+
+func (r *localIdentityIdentityProfilesResource) Delete(ctx context.Context, req resource.DeleteRequest, resp *resource.DeleteResponse) {
+	// Retrieve values from state
+	var state localIdentityIdentityProfilesResourceModel
+	diags := req.State.Get(ctx, &state)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+	httpResp, err := r.apiClient.LocalIdentityIdentityProfilesApi.DeleteIdentityProfile(config.ProviderBasicAuthContext(ctx, r.providerConfig), state.Id.ValueString()).Execute()
+	if err != nil {
```

Comment:
This should ignore errors that signify the resource doesn't exist: https://developer.hashicorp.com/terraform/plugin/framework/resources/delete#recommendations
Also the return isn't needed here